src/components/src/test/java/org/apache/jmeter/timers/poissonarrivals/PreciseThroughputTimerTest.java

```diff
@@ -68,20 +69,10 @@ public void testExactNumberOfSamples() throws Exception {
     protected ConstantPoissonProcessGenerator getConstantPoissonProcessGenerator(
             final double throughput, final int duration, long seed) {
         return new ConstantPoissonProcessGenerator(
-                new ThroughputProvider() {
-                    @Override
-                    public double getThroughput() {
-                        return throughput; // samples per second
-                    }
-                },
+                () -> throughput,
```

Comment:
Did you replace this manually?
Did you use an IDE action to automatically find and replace it?

Reply:
IntelliJ suggested it, however it was a 2 step process, first it changed it to:

```
() -> {
    return throughput; // samples per second
}
```
Then I removed the comment and it correct it to `() -> throughput`.

Reply:
In IntelliJ IDEA:
1. Help -> "Find action..."
2. "Run inspection by name..."
3. "Anonymous type can be replaced with lambda"
4. "whole project", "include test sourecs"

It produces a list of 20 or so items to be replaced.
It would be so much better if you included that change in a separate commit.
All of those changes could go under `Apply "Anonymous type can be replaced with lambda" inspection` commit.

WDYT?